### DISCUSSION TITLE
Malware hosted in Github User Attachments

### DISCUSSION THREAD

**Discussion by haydanbarry** [community/community#127770] [Repositories] [answered]
### Select Topic Area

Question

### Body

Why doesn't Github offer the option to actually contact their support team. They provide ways to report abuse for everything EXCEPT user attachments... and guess what, this part of your platform is hosting malware. Here you go... please take this down. While your at it, maybe provide some way to report these files for take down.

https://github.com/user-attachments/files/15592343/Rachel.Completed.Organizer.Season.TAX.2023.zip

**Comment by queenofcorgis** [comment, marked as answer]
Hi @haydanbarry 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. You would use the Reporting a User functionality to report the maintainers of the repo. 

- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
